mopro-ffi/tests/bindings/test_mopro.swift

```diff
@@ -6,51 +6,26 @@ let moproCircom = MoproCircom()
 let wasmPath = "./../../../../mopro-core/examples/circom/multiplier2/target/multiplier2_js/multiplier2.wasm"
 let r1csPath = "./../../../../mopro-core/examples/circom/multiplier2/target/multiplier2.r1cs"
 
-// TODO: should handle 254-bit input
-func serializeOutputs(_ int32Array: [Int32]) -> [UInt8] {
-    var bytesArray: [UInt8] = []
-    let length = int32Array.count
-    var littleEndianLength = length.littleEndian
-    let targetLength = 32
-    withUnsafeBytes(of: &littleEndianLength) {
-        bytesArray.append(contentsOf: $0)
-    }
-    for value in int32Array {
-        var littleEndian = value.littleEndian
-        var byteLength = 0
-        withUnsafeBytes(of: &littleEndian) {
-            bytesArray.append(contentsOf: $0)
-            byteLength = byteLength + $0.count
-        }
-        if byteLength < targetLength {
-            let paddingCount = targetLength - byteLength
-            let paddingArray = [UInt8](repeating: 0, count: paddingCount)
-            bytesArray.append(contentsOf: paddingArray)
-        } 
-    }
-    return bytesArray
-}
-
 do {
     // Setup
     let setupResult = try moproCircom.setup(wasmPath: wasmPath, r1csPath: r1csPath)
     assert(!setupResult.provingKey.isEmpty, "Proving key should not be empty")
 
     // Prepare inputs
-    var inputs = [String: [Int32]]()
-    inputs["a"] = [3]
-    inputs["b"] = [5]
+    var inputs = [String: [String]]()
+    let field = "21888242871839275222246405745257275088548364400416034343698204186575808495616"
+    inputs["a"] = [field]
+    inputs["b"] = ["1"]
```

Comment:
In Swift, perhaps you'd still do normal math operations on numbers, and then only do the serialization to string at the very end before calling the FFI?